fortium-qbo/app/routers/auth.py

```
@@ -132,6 +132,8 @@ async def callback(request: Request):
 
         return response
 
+    except HTTPException:
```

Ruff (0.16.1)

[error] Undefined name `HTTPException`

(F821)